### PR TITLE
workshop relevant code changed for demo

### LINKED ISSUE #1228: added demo to release

### PR DESCRIPTION
**Pull request by alonp99**

### CODE REVIEW

File: apps/backoffice-v2/src/pages/Entity/hooks/useTasks/useTasks.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ export const useTasks = ({
               {
                 id: 'nested-details-heading',
                 type: 'heading',
-                value: 'Registry information',
+                value: `Plugin (${key})`,
```
Comment (alonp99) [inline apps/backoffice-v2/src/pages/Entity/hooks/useTasks/useTasks.tsx:84]: @Omri-Levy Its a change i had to make so it wont be hard coded on the "Registry information" plugin

File: deploy/docker-compose-ex-prebuilt.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    image: ghcr.io/ballerine-io/backoffice:fintech_devcon
+    platform: linux/amd64
+    ports:
+      - 5137:80
+    depends_on:
+      - ballerine-workflow-service
+    restart: on-failure
+  ballerine-kyb-app:
+    image: ghcr.io/ballerine-io/kyb-app:fintech_devcon
+    platform: linux/amd64
```
Comment (alonp99) [inline deploy/docker-compose-ex-prebuilt.yml:13]: @pratapalakshmi its probably not a great soultion for the issue we had `no matching manifest for linux/arm64/v8 in the manifest list entries" - can you open a task to find appriopate solution?

File: examples/kyb-app/src/common/utils/request.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import ky from 'ky';
 
 export const request = ky.extend({
   prefixUrl: import.meta.env.VITE_API_URL as string,
+  timeout: 10000,
```
Comment (alonp99) [inline examples/kyb-app/src/common/utils/request.ts:5]: @chesterkmr it seems like the "/finish" request waits the all workflow to resolve his first run. it should just send the event and return. can you add a task to next the sprint?

File: examples/kyb-app/src/hooks/useActiveWorkflowQuery/useActiveWorkflowQuery.ts [hunk 4]
```diff
@@ -28,10 +28,20 @@ export const useActiveWorkflowQuery = (documentConfigurations: DocumentConfigura
     const filesOrIds = documentConfigurations.map(config =>
       traverseObjectAndPickValue(config.name, _flowData),
     );
+
+    // This is exit of useEffect
+    // Checks when all data is assigned
+    // TO DO: Rework flow
```
Comment (alonp99) [inline examples/kyb-app/src/hooks/useActiveWorkflowQuery/useActiveWorkflowQuery.ts:34]: @chesterkmr please add a task to next sprint

File: services/workflows-service/src/workflow/workflow.controller.external.ts [hunk 5]
```diff
@@ -68,6 +71,22 @@ export class WorkflowControllerExternal {
     return await this.service.getWorkflowDefinitionById(params.id);
   }
 
+  @common.Patch('/workflow-definition/:id')
+  @ApiOkResponse({ type: WorkflowDefinitionModel })
+  @swagger.ApiNotFoundResponse({ type: errors.NotFoundException })
+  @UseCustomerAuthGuard()
+  async updateWorkflowDefinition(
+    @common.Param() params: WorkflowDefinitionWhereUniqueInput,
+    @common.Body() body: unknown,
```
Comment (alonp99) [inline services/workflows-service/src/workflow/workflow.controller.external.ts:80]: Add type

File: services/workflows-service/src/workflow/workflow.service.ts [hunk 6]
```diff
@@ -293,6 +296,18 @@ export class WorkflowService {
     return await this.workflowDefinitionRepository.findById(id, args);
   }
 
+  async updateWorkflowDefinitionById(
+    id: string,
+    newDefinition: unknown,
```
Comment (alonp99) [inline services/workflows-service/src/workflow/workflow.service.ts:301]: add type

File: services/workflows-service/src/workflow/workflow.service.ts [hunk 7]
```diff
@@ -1379,7 +1394,7 @@ export class WorkflowService {
         machineContext: workflowRuntimeData.context,
         state: workflowRuntimeData.state,
       },
-      extensions: workflowDefinition.extensions,
+      extensions: workflowDefinition.definition.extensions,
```
Comment (Omri-Levy) [inline services/workflows-service/src/workflow/workflow.service.ts:1397]: `extensions` exists at the root of `WorkflowDefinition` in `schema.prisma`.

**Pull request closed without merging**